File: solutions/python/isogram/2/isogram.py
def is_isogram(string: str) -> bool:
    """It evaluates whether a given word is an isogram
    
    :param string: The word to be evaluated.
    :return: bool - Whether the word is an isogram or not.
    """
    
    clean = string.replace("-", "").replace(" ", "").lower()
    return len(clean) == len(set(clean))

File: solutions/javascript/isogram/2/isogram.js
/**
* It takes in a word and evaluates whether it is an isogram
*
* @param {string} word The word to be evaluated.
* @returns {boolean} Whether the word is an isogram or not.
*/
export const isIsogram = word => word.replace(/[- ]/g, "").length == new Set(word.replace(/[- ]/g, "").toLowerCase()).size;
